src/pages/components/feature/FeatureCard.js

```diff
@@ -16,8 +16,8 @@ const FeatureCard = ({ item, index, length }) => {
     if (index < length - 1) {
       return (
         <div
-          className={`absolute top-[155px] -z-10 hidden md:left-[270px] md:block 
-            lg:left-[245px] xl:left-[400px] xl:top-[270px] 2xl:left-[575px] 2xl:top-[380px]
+          className={`absolute top-[155px] -z-10 hidden sm:hidden md:top-[286px] md:left-[369px] md:block 
+            lg:left-[401px] xl:left-[355px] xl:top-[231px] 2xl:left-[457px] 2xl:top-[307px]
```

Comment:
```suggestion
          className={`absolute top-[155px] -z-10 hidden md:top-[286px] md:left-[369px] lg:block
            lg:left-[401px] xl:left-[355px] xl:top-[231px] 2xl:left-[457px] 2xl:top-[307px]
```
 I think you can remove the sm:hidden and change the md:block to lg:block
so that from mobile devices to tablet it won't display, and only left is the positioning of the paw trails in laptop devices, make sure to test it.

and you can remove the md: '920px' in the tailwind config.